packages/common/src/deprecated/getTableIds.ts

```diff
@@ -0,0 +1,6 @@
+import { TableId } from "./TableId";
+
+/** @deprecated Use `tableIdToHex` and `hexToTableId` instead. */
```

Comment:
this comment seems wrong; is `getTableIds` deprecated and if so what is the recommended alternative?

Reply:
`getTableIds` is used by the snap sync module in network, which expects `TableIds[]` so it didn't seem worth it to migrate